docs/source/overview.md

```diff
@@ -0,0 +1,81 @@
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![Sample linked data](./_static/img/mergedlinkeddata.svg)
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center">
+
+_Linked data about a city from three different sources: the city's traffic authority, a map from a city guide, and the university registry. Each data source is represented using a different color and column._
+
+</figcaption>
+
+</figure>
+
+
+Moreover, in SimPhoNy, all linked data is based on [ontologies](https://en.wikipedia.org/wiki/Ontology_(information_science)), which give **meaning** to the data. Specifically, SimPhoNy works with ontologies based on the [Web Ontology Language](https://en.wikipedia.org/wiki/Web_Ontology_Language), making the data compatible with the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web).
+
+### Fetch data from a database, run a simulation and immediately store the results
+
+Ontology-based linked data is not only well suited for the interoperability of data, but also of software tools. In SimPhoNy, one can instantiate individuals from special ontology classes called _wrappers_. These wrappers are both an ontology individual, and a software interface between the core of SimPhoNy (ontology based) and external software tools. We have already developed a few wrappers for some software tools, but you can even [develop your own](./wrapper_development.md)!
+
+As a SimPhoNy user, you can see the data stored in the external software tools transparently as ontology individuals through the wrappers. In this way, moving data between different software tools becomes as simple as moving or copying it from one wrapper to another.
+
+For example, linked data stored in a SQLite database can be used to run a simulation just by adding the ontology individuals contained in the SQLite wrapper to the Simulation Engine wrapper. Similarly, the ontology individuals representing the results can be simply added back into the database wrapper.
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![How wrappers work](./_static/img/wrappers.svg)
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center"></figcaption>
+
+</figure>
+
+At this point, the results could be fetched again and for example, visualized with the help of a plotting library.
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![Toy example of simulation results](./_static/img/trafficsim.svg)
+
+<img src="./_static/img/overview_slider.png" width="350px">
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center">
+
+<br>
+
+_These results are just a toy example, they do not come from any real nor factually correct simulation._
+
+</figcaption>
+
+</figure>
+
+### Couple simulation engines easily
```

Comment:
The video of Pablo could be added here.

Reply:
I'll think for a while about this and [Pablo's comment](https://github.com/simphony/docs/pull/121/#pullrequestreview-632996879). If, before the sprint retrospective, I come up with an simple reformulation that does not make the review too heavy and overcomes the issue that the example in the video has nothing to do with the one on the overview, then I'll make changes. If not this will have to be left for next time.

Reply:
@yoavnash @pablo-de-andres Seems feasible, check [f57f8ed](https://github.com/simphony/docs/pull/121/commits/f57f8edfc6300cf02a303a4e69c959db02d5649e) or screenshot below. I hope the whole thing does not get confusing because I changed the example.

![grafik](https://user-images.githubusercontent.com/43052541/114365784-13c6a400-9b7b-11eb-9c2d-56dd972b03ae.png)